modin/numpy/test/test_array.py

```diff
@@ -18,11 +18,28 @@
 import modin.numpy as np
 
 
-@pytest.mark.parametrize("size", [100, (2, 100), (100, 2), (1, 100), (100, 1)])
+@pytest.mark.parametrize(
+    "size",
+    [
+        100,
+        (2, 100),
+        (100, 2),
+        (1, 100),
+        (100, 1),
+        (100, 100),
+        (6, 100),
+        (100, 6),
+        (100, 7),
+        (7, 100),
+    ],
+)
 def test_repr(size):
+    prev_threshold = numpy.get_printoptions()["threshold"]
+    numpy.set_printoptions(threshold=50)
```

Comment:
this should actually be configured within pytest fixture... or at least wrap it in `try..finally`, as otherwise when e.g. `assert` would fire the threshold would not be restored back

Reply:
Sounds good to me!